src/components/Icon/Add.tsx

```diff
@@ -1,28 +1,34 @@
 import * as React from "react";
 import { IconProps } from "./Icon";
 
-const SvgAdd = (props: IconProps) => (
+const SvgAddMetamask = (props: IconProps): React.ReactElement => (
```

Comment:
We need to rebase this, this stuff should definitely not be here

Reply:
Has been fixed with the merge from main here 491a0aaad624b1fe1ba6b5d12f09cdff0119c43b